CDP4ShellDialogs/ViewModels/PluginManagerViewModel.cs

```diff
@@ -46,7 +46,12 @@ public PluginManagerViewModel(IAppSettingsService<T> appSettingsService)
             {
                 ChangeTrackingEnabled = true
             };
-            this.Plugins.ItemChanged.Subscribe(_ => { this.IsDirty = true; });
+            this.Plugins.ItemChanged.Subscribe(
```

Comment:
What I am saying is that you do not need to react to whether the row is dirty or not, you can directly react to the `IsPluginEnabled` property:

```
this.Plugins.ItemChanged
    .Where(x => x.PropertyName == "IsPluginEnabled")
    .Select(x => x.Sender)
    .Subscribe(x => {
        this.IsDirty = true;
    });
```

https://reactiveui.net/docs/handbook/obsolete/collections/reactive-list